internal/metrics/prometheus.go

```diff
@@ -129,20 +161,43 @@ func NewCtrlFuncMetrics() *CtrlFuncMetrics {
 		[]string{SuccessKey, ProtocolKey, DataplaneKey},
 	)
 
+	controllerMetrics.ConfigPushSuccessTime = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: MetricNameConfigPushSuccessTime,
+			Help: fmt.Sprintf("The time of the last successful configuration push. "+
+				"`%s` describes the dataplane that was the target of the configuration push.",
+				DataplaneKey,
+			),
+		},
+		[]string{DataplaneKey},
```

Comment:
```suggestion
		[]string{DataplaneKey, ProtocolKey},
```
Seems the missing `protocol` label caused all the panics in tests.

Reply:
There's no obvious reason to include protocol for these (there arguably wasn't any reason to include it with the push count metric either--it doesn't vary within a given deployment), so stripped it from their recorder functions instead (it got unintentionally copy-pasted in from others).